app/routes/poll-network.tsx

```diff
@@ -40,28 +45,47 @@ export default function () {
   useEffect(() => {
     if (fetcher.data) {
       setStatus(fetcher.data)
+      localStorage.setItem(LS_KEY, JSON.stringify(status))
```

Comment:
I think this is going to cause an outdated version (the previous value) of status in `localStorage`, won't it? Should this be `JSON.stringify(fetcher.data)`?